src/BSD-Source-beginning-file.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="BSD-Source-beginning-file"
+            name="BSD Source Code Attribution - beginning of file variant"
+            listVersionAdded="3.23">
+      <crossRefs>
+         <crossRef>https://github.com/lattera/freebsd/blob/master/sys/cam/cam.c#L4</crossRef>
+      </crossRefs>
+      <notes>
+         This license is mostly the same as BSD-Source-Code but adds includes
+         a specific requirement requiring retention of the copyright notice
+         "immediately at the beginning of the file".
+      </notes>
+    <text>
+      <copyrightText>
+         <p>Copyright (c) 1997 Justin T. Gibbs.
+            <br/>All rights reserved.
+      </p>
```

Comment:
This is just an example of the copyright text, right? Done we have generic year and holder text we try to use?

Reply:
Right. This is for the sake of completeness in examples. But for matching, it is ignored. See https://spdx.github.io/spdx-spec/v2.3/license-matching-guidelines-and-templates/#b11-copyright-notice

Reply:
@bsdimp - we have made the copyright notice generic in some of the more "popular" license or when someone has asked to do so, but we have not made a hard rule on this. I think many of the licenses being found in Fedora at this point are quite old, so not sure it matters (and in this case, we probably would hope others don't re-use this license!)

Reply:
> @bsdimp - we have made the copyright notice generic in some of the more "popular" license or when someone has asked to do so, but we have not made a hard rule on this. I think many of the licenses being found in Fedora at this point are quite old, so not sure it matters (and in this case, we probably would hope others don't re-use this license!)

OK. Works for me then. I'm otherwise happy with this. Totally agree with 'others not using it'. I've been talking with Justin and a few other people who have this license in FreeBSD to persuade them to switch to something more standard. They are open to the idea, but the devil is in the details.